Document/MediaManager.php

```diff
@@ -34,7 +34,7 @@ public function save($entity, $andFlush = true)
             $entity->setProviderName(func_get_arg(2));
         }
 
-        if ($andFlush && is_bool($andFlush)) {
+        if (is_bool($andFlush)) {
```

Comment:
I don't understand.

Your condition just check if it's a boolean, never-mind if it's `true` or `false`.

I don't think this is the wanted behavior.

Ping @rande